### PR TITLE
feat: Add flag to exclude purchased artworks from My Collection query

### PR DESCRIPTION
**Pull request by olerichter00**
Addresses [CX-1868]

## Description

Adds a flag to exclude purchased artworks from "My Collection" query. This is necessary to only show artworks that have been automatically added to "My Collection" to specific users until we release the feature for all users.

[CX-1868]: https://artsyproduct.atlassian.net/browse/CX-1868?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/schema/v2/me/myCollection.ts [hunk 1]
```diff
@@ -67,13 +67,23 @@ export const MyCollection: GraphQLFieldConfig<any, ResolverContext> = {
         },
       }),
     },
+    excludePurchasedArtworks: {
+      type: GraphQLBoolean,
```
Comment (olerichter00) [inline src/schema/v2/me/myCollection.ts:71]: @MounirDhahri feel free to commit and add a default value here:

```suggestion
      type: GraphQLBoolean,
      defaultValue: false,
```

Comment (MounirDhahri) [reply src/schema/v2/me/myCollection.ts:71]: 🙏

**Review by MounirDhahri — APPROVED** [review]
👍👍👍

**Comment by olerichter00** [comment]
#squashongreen

File: src/schema/v2/me/myCollection.ts [hunk 2]
```diff
@@ -67,13 +67,24 @@ export const MyCollection: GraphQLFieldConfig<any, ResolverContext> = {
...
+      defaultValue: false,
```
Comment (brainbicycle) [inline src/schema/v2/me/myCollection.ts:72]: should this be true instead for backwards compatibility?

**Comment by MounirDhahri** [comment]
😬 good catch

**Pull request merged**